### PR TITLE
Add shortname lws for API & sort out build image command

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it

Commits:
1. We can now build customized image by specifying `IMAGE_REGISTRY`
2. Reduce controller resource requests from 2C2Gi to 1C1Gi for local experiment
3. Add shortName `lws` for leaderWorkerSet, and remove folder `/artifacts`, which is ignored by `.gitignore` now.

#### Which issue(s) this PR fixes

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Add shortName `lws` for leaderWorkerSet API object.
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +14,13 @@ endif
 # tools. (i.e. podman)
 CONTAINER_TOOL ?= docker
 
+GIT_TAG ?= $(shell git describe --tags --dirty --always)
+# Image URL to use all building/pushing image targets
+IMAGE_REGISTRY ?= gcr.io/k8s-staging-lws
```
Comment (kerthcet) [inline Makefile:19]: We may need an official image address.

Comment (ahg-g) [reply Makefile:19]: That sounds good, I guess we need a PR similar to https://github.com/kubernetes/k8s.io/pull/5144

**Comment by ahg-g** [comment]
/label tide/merge-method-squash
/lgtm
/approve

Can we please update the description to indicate that this PR is part of https://github.com/kubernetes-sigs/lws/issues/29

**Comment by kerthcet** [comment]
> /label tide/merge-method-squash /lgtm /approve
> 
> Can we please update the description to indicate that this PR is part of #29

Done

**Pull request merged**